core/__tests__/modules/configWriter.ts

```diff
@@ -364,11 +372,14 @@ describe("modules/configWriter", () => {
         });
         expect(ConfigWriter.getLockKey(configObject)).toEqual(null);
         process.env.GROUPAROO_RUN_MODE = undefined;
+        rebuildConfig();
       });
       test('returns "code:config" when in start mode', async () => {
         process.env.GROUPAROO_RUN_MODE = "cli:start";
+        rebuildConfig();
         expect(ConfigWriter.getLockKey(configObject)).toEqual("config:code");
         process.env.GROUPAROO_RUN_MODE = undefined;
+        rebuildConfig();
```

Comment:
Do you need to run this since it's the end of the method? Maybe we should just call it in `beforeEach`?